devimint/src/main.rs

```diff
@@ -1040,7 +1040,14 @@ async fn run_ui(
                 Ok(TcpStream::connect(server_addr).await.is_ok())
             })
             .await?;
-            info!(LOG_DEVIMINT, "Started ui/api on http://{server_addr}");
+            let url_protocol = match kind {
+                RunUiKind::Old => "http://",
+                RunUiKind::New => "ws://",
+            };
```

Comment:
Could this log be moved inside the UI server impl, close to where the server is actually started? That would make more sense imo since it's more apparent which protocol the server started is running in that context and you wouldn't need to match here.

Reply:
Please see updated code.